### ISSUE TITLE
Upgrade end point integrations

### ISSUE THREAD

**Issue by btopro** [elmsln/lrnwebcomponents#127]
The platform integrations for ES modules need to be fixed to account for node_modules and JS based module finding instead of HTML Imports.
Prerequisites:
- [x] document build routine appropriately for use in both an existing theme and stand alone
- [x] get a good package.json to ship w/ all of them which *just references something else* (included below)
- [x] `@lrnwebcomponents/baseline-build-hax` - create a project to publish which basically just ensures everything else is referenced correctly for the baseline so that we don't have to keep updating bower.json in each project. this project will include wysiwyg-hax / cms-hax / hax-body etc but also have listed as real dependencies the things we typically pull in for demonstrative purposes. ~~Building and keeping the built files in a repo MAY allow us to serve them correctly from unpkg cdn~~ (CDN not possible until https://github.com/unpkg/unpkg.com/issues/108 and https://github.com/unpkg/unpkg.com/issues/151)
- [x] try moving babel translations into own files and only importing when platform requires it (ES5 stuff as the ES6 targets shouldn't need the definitions)

Targets:
- [x] GravCMS
- [x] HAXCMS
- [x] Drupal 8
- [x] Drupal 6
- [x] Backdrop
- [x] Drupal 7 (reduce scope of D7 version drastically)
- [x] ELMS:LN

**Comment by btopro** [comment]
start at least
```
{
  "name": "simple-build-for-hax-named-whatever-you-want",
  "version": "0.0.37",
  "description": "",
  "repository": {
    "type": "git",
    "url": "git@github.com/elmsln/lrnwebcomponents.git"
  },
  "main": "baseline-build-hax.js",
  "module": "baseline-build-hax.es6.js",
  "umd": "baseline-build-hax.umd.js",
  "amd": "baseline-build-hax.amd.js",
  "scripts": {
    "test": "../../node_modules/.bin/wct --configFile ../../wct.conf.json node_modules/@lrnwebcomponents/baseline-build-hax/test/",
    "start":"yarn run dev","build":"../../node_modules/.bin/gulp && ../../node_modules/.bin/rollup -c && ../../node_modules/.bin/prettier --ignore-path ../../.prettierignore --write '**/*.{js,json}'","dev":"concurrently --kill-others \"yarn run watch\" \"yarn run serve\"","watch":"../../node_modules/.bin/gulp dev","serve":"polymer serve --npm --module-resolution=node --open","lighthouse":"../../node_modules/.bin/gulp lighthouse"
  },
  "author": {
    "name": "btopro"
  },
  "license": "Apache-2.0",
  "dependencies": {
    "@lrnwebcomponents/baseline-build-hax": "^0.0.37"
  },
  "devDependencies": {
    "@lrnwebcomponents/deduping-fix":"0.0.37",
    "@polymer/iron-demo-helpers":"3.0.2",
    "@polymer/iron-component-page":"github:PolymerElements/iron-component-page",
    "concurrently":"4.1.0",
    "gulp-babel":"8.0.0",
    "gulp-sourcemaps":"2.6.4",
    "lodash":"4.17.11",
    "polymer-cli":"1.9.1",
    "polymer-build":"3.1.1",
    "wct-browser-legacy":"1.0.2"
  },
  "private": false,
  "publishConfig": {
    "access": "public"
  },
  "keywords": [
    "web-components",
    "html"
  ]
}
```

**Comment by btopro** [comment]
elmsln hasn't switched over yet bc of having to fix up some apps but it's working in all end points
